config/sources/families/include/meson64_common.inc

```diff
@@ -22,9 +22,16 @@ GOVERNOR=${GOVERNOR:-ondemand}
 
 case $BRANCH in
 
+	legacy)
```

Comment:
Legacy kernel 4.9 was recently removed. Any reason for keeping it?

Reply:
This shit (BSP AML) in principle should not be in the build system.